### DISCUSSION TITLE
PRYCAM

### DISCUSSION THREAD

**Discussion by andreacataldi1981** [indomus/forum#2512] [GENERALE - Discussioni generiche]
buongiorno 
ho acquistato l'oggetto PRYCAM, ho contattato la ditta che mi ha detto che stanno pensando ad una versione beta per collegarlo ad Homeassit.
ma non riesco a settare il settore.
mi hanno detto che hanno configurato con Mqtt riferito al mio IP, e mi hanno dato il topic da subsrivere, ma mi ha stato sconusciuto

qualcuno mi puo dare una mano

questo è il testo della loro email:

Per ricevere le letture ogni secondo deve fare subscribe al topic 
"phome/PH010000800000000000XXXX/tele"

grazie

**Comment by indomus** [comment]
https://indomus.it/componenti/message-queue-telemetry-transport-mqtt/
